pkg/types/aws/defaults/platform.go

```diff
@@ -1,14 +1,19 @@
 package defaults
 
 import (
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/aws"
 )
 
 var (
-	defaultMachineClass = map[string][]string{
-		// Example region default machine class override:
+	defaultMachineClassAMD64 = map[string][]string{
+		// Example region default machine class override for AMD64:
 		// "ap-east-1":      {"m5", "m4"},
 	}
+	defaultMachineClassARM64 = map[string][]string{
+		// Example region default machine class override for ARM64:
+		// "us-east-1":      {"m6g", "m6gd"},
+	}
```

Comment:
```suggestion
	defaultMachineClass = map[types.Architecture]map[string][]string{
		types.ArchitectureAMD64: {
			// Example region default machine class override for AMD64:
			// "ap-east-1":      {"m5", "m4"},
		},
		types.ArchitectureARM64: {
			// Example region default machine class override for ARM64:
			// "us-east-1":      {"m6g", "m6gd"},
		},
	}
```

Reply:
commited this and the below change into a single commit